docs/api.md

```diff
@@ -998,12 +998,14 @@ Emitted for every server message which appears on the Action Bar.
 
  * `jsonMsg` - unmodified JSON message from the server
 
-#### "message" (jsonMsg)
+#### "message" (jsonMsg, position)
 
 Emitted for every server message, including chats.
 
  * `jsonMsg` - unmodified JSON message from the server
 
+ * `positon` - position of Chat message
```

Comment:
can you add here what is the meaning of this value?
Plus a warning that it isn't available in versions before 1.16?

Reply:
![image](https://user-images.githubusercontent.com/29885012/95587296-2d5b2780-0a42-11eb-9e87-2a07e862a05b.png)

https://wiki.vg/Chat

Do you have any suggestions for the meaning of this value?

Reply:
no, that's why I think if it is exposed it should be well documented
Most likely we don't want to expose it directly but add some abstraction on top so it doesn't break in the next minecraft release.